src/bp/ui-admin/src/telemetry.ts

```diff
@@ -112,84 +113,64 @@ export function checkInfoReceived() {
   return !_.includes(info, '')
 }
 
-export function getServerFeedback() {
-  const pkgStr = window.localStorage.getItem('packageToSend')
-  let packages: Array<object> = []
-  if (pkgStr !== null) {
-    packages = JSON.parse(pkgStr)
-  }
-  return packages
-}
+export async function sendServerPackage() {
+  try {
+    const packages = (await api.getSecured().get(serverUrl)).data
 
-export function feedback(pkg) {
-  axios
-    .post(serverUrl, pkg, corsConfig)
-    .then(res => {
-      const packages = getServerFeedback()
-      if (packages.indexOf(pkg) !== -1) {
-        packages.splice(packages.indexOf(pkg), 1)
-        window.localStorage.setItem('packageToSend', JSON.stringify(packages))
-      }
-      console.log(res)
-    })
-    .catch(err => {
-      const packages = getServerFeedback()
-      if (packages.indexOf(pkg) === -1) {
-        packages.push(pkg)
-        window.localStorage.setItem('packageToSend', JSON.stringify(packages))
+    console.log('packages got', packages)
+
+    const feedback = { events: [], status: '' }
+
+    if ((packages !== undefined && _.has(packages, 'url'), _.has(packages, 'events'))) {
+      const url = packages.url
+      const events = packages.events
+
+      let i = 0
+      for (const obj of events) {
+        if (typeof obj === 'string') {
```

Comment:
does that really happens ? I'm pretty sure axios parses the json for you, you should have js objects here not json strings, if not then there might be something wrong in the backend...

that whole loop becomes useless as it's only for parsing